torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -1972,9 +1975,9 @@ Tensor svd_backward(const std::vector<torch::autograd::Variable> &grads, const T
   // https://arxiv.org/abs/1909.02659
   if (self.is_complex() && gu.defined()) {
     // computes L = Identity.mul(uh @ gu)
-    Tensor L = at::matmul(uh, gu).diagonal(0, -2, -1).diag_embed(0, -2, -1);
-    L = L - L.conj().transpose(-2, -1);
-    Tensor imag_term = 0.5 * at::matmul(at::matmul(at::matmul(u, L), sigma_mat_inv), vh);
+    Tensor L = at::matmul(uh, gu).diagonal(0, -2, -1);
+    L = L - L.conj();
+    Tensor imag_term = 0.5 * at::matmul(u * (L * sigma_inv).unsqueeze(-2), vh);
```

Comment:
nit: Could it be better to replace `L` with just `2 * L.imag()` and use the float-float `mul` in `L * sigma_inv` instead of the float-complex one, then multiply by the imaginary unit in the most efficient way...

Reply:
That's a simple change that might give better performance, so I think it's worth adding it to this PR.

Reply:
Also you don't need to do the `2*` if you remove the `0.5*` right? :D

Reply:
Yes, so one more reason to follow the not.